.github/workflows/test_inc.yml

```diff
@@ -30,8 +30,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install cmake
+        pip install py-cpuinfo
+        pip install torch==2.1.0 torchaudio==2.1.0 torchvision==0.16 --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
no, because there is a library of libqbits.so in ITREX 1.3.2, and it is built with torch2.1.0, if we install the latest torch version, it will lead to an issue.

Reply:
Could you extend on what type of issue ? We should raise an error for unsupported version of torch then

Reply:
The error is: libqbits.so: undefined symbol: _ZNR5torch7Library4_defEON3c106eitherINS1_12OperatorNameENS1_14FunctionSchemaEEEONS_11CppFunctionE